api/views.py

```diff
@@ -1,7 +1,23 @@
 from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from rest_framework.parsers import JSONParser
+from lyrics.models import Lyric
+from .models import LyricSerializer
 
 
 def test(request):
     template = ""
     context = {}
     return render(request=request, template_name=template, context=context)
+
+
+@api_view(["GET"])
+def lyric_detail(request, pk):
+    try:
+        lyric = Lyric.objects.get(pk=pk)
+    except Lyric.DoesNotExist:
+        return HttpResponse(status=404)

```

Comment:
that line of code should not exist. We're writing apis, as such any response should be a JsonResponse. So basically, that line should be `Response(data={"error": "Lyric not found"}, status=status.HTTP_404_NOT_FOUND)`

Reply:
is this resolved?

Reply:
yeah